### PR TITLE
MTDSIM -88 Refactor components folder

### PR DESCRIPTION
**Pull request by nadunvindy**
Refactor files in components folder

### CODE REVIEW

File: server/mtdnetwork/component/host.py [hunk 1]
```diff
@@ -291,7 +293,6 @@ def get_services_from_list(self, list):
                 list of shortest path to target node (including target node)
 
         """
-        target = list.pop(len(list) - 1)
```
Comment (nicodeas) [inline server/mtdnetwork/component/host.py:294]: why is this deleted?

File: server/mtdnetwork/component/host.py [hunk 2]
```diff
@@ -675,7 +681,7 @@ def get_path_from_exposed(self):
                 if path_len < shortest_distance:
                     shortest_distance = path_len
                     shortest_path = path
-            except:
+            except Exception:
                 pass
```
Comment (nicodeas) [inline server/mtdnetwork/component/host.py:685]: can you  please throw an error or log that an error has occured here.

File: server/mtdnetwork/component/network.py [hunk 3]
```diff
@@ -657,7 +661,7 @@ def get_shortest_distance_from_exposed_or_pivot(
 
                 if path_len < shortest_distance:
                     shortest_distance = path_len
-            except:
+            except Exception:
```
Comment (nicodeas) [inline server/mtdnetwork/component/network.py:664]: log error

**Comment by nicodeas** [comment]
search up how to set python log level to error with the python logging module

File: server/mtdnetwork/component/host.py (hunk 1 above)
Comment (nadunvindy) [reply server/mtdnetwork/component/host.py:294]: variable was unused

**Comment by benvarian** [comment]
is this ready to be looked at

**Review by nicodeas — APPROVED** [review]
